docs/source/routing/graphos-reporting.mdx

```diff
@@ -388,8 +388,53 @@ telemetry:
             send: false
 ```
 
+#### Enabling Extended Error Reporting
+
+<PreviewFeature />
+
+Enable richer error reporting via `preview_extended_error_metrics` and `redaction_policy` options.
+
+When enabled, the Router will send metrics to Studio
+with additional attributes including the `service` and `code` found in GraphQL error extensions (`errors[].extensions.service` and `errors[].extensions.code`).
+
+Additional diagnostic capabilities available in Studio now support viewing errors by `service` / `code`. The `service` dimension refers to the subgraph or connector where the error originated from.
+The `code` refers to the specific type of error that was raised by the Router or federated subgraphs / connectors.
+
+```yaml title="router.yaml"
+telemetry:
+  apollo:
+    errors:
+      preview_extended_error_metrics: enabled # (default: disabled)
+      subgraph:
+        all:
+          # By default, subgraphs should report errors to GraphOS
+          send: true # (default: true)
+          redaction_policy: extended # (default: strict)
+        subgraphs:
+          account: # Override the default behavior for the "account" subgraph
+            send: false
+```
+
+<Note>
+
+At scale, this feature is known to hit cardinality limitations in the OTel reporting agent. When this happens, some of the extended metrics attributes may no longer
+be visible in Studio. In the case of cardinality warnings in logs, adjusting the Apollo batch processing configuration to send reports more frequently can help to alleviate this.
+
+Additionally, this feature may increase the Router's memory usage profile. Similarly, lowering the `scheduled_delay` can help to alleviate that as well. See the example below.
+
+</Note>
+
+```yaml title="router.yaml"
+telemetry:
+  apollo:
+    batch_processor:
+      scheduled_delay: 100ms # default is 5s
+```
+
+### Plugin Authors
```

Comment:
Hmm, this new H3 confuses me. Was there meant to be more content here, or is it just the note below? If so, let's remove the `<Note>` formatting and perhaps rename the section for clarity.  (e.g. "Accessing operation IDs in plugins"

Also this info was previously nested in the [`errors` section](https://www.apollographql.com/docs/graphos/routing/graphos-reporting#errors). Is an H3 under `advanced configuration` the best place for it?